apps/docs/src/components/navigation/navigation.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import {ArrowUpRightIcon, TriangleDownIcon} from '@primer/octicons-react'
+import styles from './navigation.module.css'
+import {ActionList, ActionMenu, Button} from '@primer/react'
+
+const navItems = [
+  {href: 'https://primer.style/', label: 'Product UI'}, // TODO: update to https://primer.style/product when we launch the new docs site
+  {href: '/', label: 'Brand UI'},
+  {href: 'https://primer.style/octicons', label: 'Octicons'},
+  {href: 'https://primer.style/guides/accessibility', label: 'Accessibility'}, // TODO: update to https://primer.style/accessibility when we launch the new docs site
+  {href: 'https://brand.github.com/', label: 'Brand Toolkit', external: true},
+]
+
+export function Navigation() {
+  const activeItem = navItems.find((item) => item.label === 'Brand UI')
+  const otherItems = navItems.filter((item) => item.label !== 'Brand UI')
+  const sortedNavItems = activeItem ? [activeItem, ...otherItems] : otherItems
```

Comment:
Should we use consistent ordering given that this is a navigation menu shared across all docs sites?

E.g. in product docs, the product link comes first, but in brand docs the brand link is first. I'd expect them to stay the same so perhaps they don't need to be sorted at all?

![Screenshot 2024-12-05 at 14 29 31](https://github.com/user-attachments/assets/3cc750ac-54ad-47d0-8dd5-31899341fd39)


![Screenshot 2024-12-05 at 14 26 11](https://github.com/user-attachments/assets/ee284cd8-060e-42d8-9385-c23ab8cc23b0)

Reply:
Forgot to change this when we changed the design of the top nav. Pushing up a fix.

Reply:
Agreed - @mperrotti and I actually spoke about this last night. Will fix before it gets merged.